Library/Formula/llvm.rb

```diff
@@ -129,9 +131,19 @@ def install
         # Let's use clang to build libcxx. It fails with gcc.
         ENV["CC"] = bin/"clang"
         ENV["CXX"] = bin/"clang++"
+        libcxx_args = %w[
+          -DLIBCXX_CXX_ABI=libsupc++
+        ]
+        libcxx_args << "-DLIBCXX_LIBSUPCXX_INCLUDE_PATHS=#{`echo | g++ -Wp,-v -x c++ - -fsyntax-only 2>&1|while read i; do [ -f "$i"/cxxabi.h ] && echo -n $i && echo -n ';'; [ -f "$i"/bits/c++config.h ] && echo -n $i && echo -n ';'; done`}"
```

Comment:
Can you relate me to some info how it should be properly wrapped? As I have mentioned, I have no knowledge of Ruby idioms since my whole experience with the language is through Homebrew Formulas :)